frontend/src/scenes/session-recordings/player/PlayerController.tsx

```diff
@@ -153,6 +163,9 @@ export function PlayerController({
                                     >
                                         Autoplay enabled
                                     </LemonButton>
+                                    <LemonButton status="stealth" fullWidth onClick={() => onAddToPlaylist()}>
+                                        Add recording to static playlist
```

Comment:
I'm thinking also copy-wise we should maybe call this "save recording" with some info about static playlists. Bit more user-friendly than our own, more-details jargon